Dockerfile

```diff
@@ -1,7 +1,21 @@
 FROM richarvey/nginx-php-fpm:1.10.3
+USER root
 
 ENV WEBROOT="/var/www/html/public/"
+ENV PHP_CATCHALL="1"
```

Comment:
Don't set this to 1 in Dockerfile because this will cause the production build to show errors